plugins/inputs/t128_metrics/t128_metrics_test.go

```diff
@@ -517,16 +529,16 @@ func TestLoadsFromToml(t *testing.T) {
 
 	plugin := &plugin.T128Metrics{}
 	exampleConfig := []byte(`
-	base_url = "example/base/url/"
-	unix_socket = "example.sock"
-	max_simultaneous_requests = 15
-	timeout = "500ms"
+	base_url                    = "example/base/url/"
```

Comment:
This isn't exactly what I meant, but it does look better. I was saying everything in the string be tabbed out one more so that it doesn't mix in as a variable. NBD though

Reply:
I just realized that with syntax highlighting, it would be a lot more apparent. I'm just reviewing on Github so that is why it took me a sec

Reply:
Gocha, yeah I misunderstood. I didn't even think about it exactly because of my syntax highlighting 🙂